main.go

```diff
@@ -171,22 +182,17 @@ func main() {
 		{
 			orgs.Use(api.OrganizationMiddleware)
 
-			orgs.GET("/:orgid/applications", api.GetApplications)
-			orgs.POST("/:orgid/applications", api.CreateApplication)
-			orgs.GET("/:orgid/applications/:id", api.ApplicationDetails)
-			orgs.DELETE("/:orgid/applications/:id", api.DeleteApplications)
-
-			orgs.GET("/:orgid/catalogs", api.GetCatalogs)
-			orgs.PUT("/:orgid/catalogs/update", api.UpdateCatalogs)
-			orgs.GET("/:orgid/catalogs/:name", api.CatalogDetails)
+			orgs.GET("/:orgid/spotguides", api.GetSpotguides)
+			orgs.PUT("/:orgid/spotguides", api.SyncSpotguides)
+			orgs.POST("/:orgid/spotguides", api.LaunchSpotguide)
+			orgs.GET("/:orgid/spotguides/*name", api.GetSpotguide)
```

Comment:
Why is `name` parameter optional?

I'd add `HEAD` as well to for verifying if a spotguide exists or not.